package.json

```diff
@@ -40,7 +40,7 @@
     "react-router-dom": "6.21.3",
     "react-useportal": "1.0.19",
     "serve": "14.2.1",
-    "vanilla-framework": "4.7.0",
+    "vanilla-framework": "4.8.0",
```

Comment:
@edlerd @mas-who Please note that this update comes with breaking changes that need to be addressed - e.g. the colour of the information icon is now blueish by default, so if you open the demo link up there you will see things like this:

![image](https://github.com/canonical/lxd-ui/assets/56583786/c6218d31-309a-4ec4-b773-5d771de43e99)

Or this:

![image](https://github.com/canonical/lxd-ui/assets/56583786/d8829813-4cf2-47f2-9ff9-a467cb7f7291)

I recommend reviewing the items in the changelog carefully, to make sure that all the possible breaking changes are addressed, before merging it. :)